### PR TITLE
Hello-Microverse Project Features-branch

### PR DESCRIPTION
**Pull request by Aidan-bn**
#Setting up Hello-Microverse Project

Create a new GitHub repo.
Create a new branch according to the GitHub flow rules.
Add a `.gitignore ` file.
Set up a linter for` HTML and CSS` and ensure any files which are not relevant are ignored by git.
Add descriptive `README.md` file 
Add 3 files:
A .html file that includes a header with the text "`Hello Microverse!`". Use h1 tag for that.
A .css file that includes one class with styles for your header (with my favorite color)
 A test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (tufoinnkuo10) [inline index.html:6]: - [ optional] please link your style to your HTML file so as to make it functional

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Aidan-bn 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated


## Required Changes ♻️♻️
Check the comments under the review
- [ required] please give a descriptive pull request title so from it, you can know what the pull request is about.

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (Aidan-bn) [reply index.html:6]: Done

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Aidan-bn ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 

 ## Required Changes ♻️

- [ ] While you did address the previous reviewer's requirement, you didn't do so properly. `Features` is not a particularly descriptive PR title. Please change your PR title to something that encapsulates the content found in this branch

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @Aidan-bn , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**